assets/stylesheets/foreground.css

```diff
@@ -713,13 +714,13 @@ div.mw-htmlform-field-HTMLCheckField div.mw-label { height:0; }
 
 
 /* Jamie's Header Hider */
```

Comment:
Been kind of wondering if this should be default css or allow the user to make the choice of whether to show the title attributes on the main page or use their MediaWiki:Foreground.css or Common.css to hide it.